.travis.yml

```diff
@@ -8,6 +8,9 @@ before_install:
 - brew update
 - bundle update
 - echo -e "machine github.com\n  login $CI_USER_TOKEN" >> ~/.netrc
+- openssl aes-256-cbc -K $encrypted_d59c41fdd72b_key -iv $encrypted_d59c41fdd72b_iv
```

Comment:
We've decided to use ENV vars for secrets instead of committing encrypted files to the repo